### DISCUSSION TITLE
Page Rules continue to not be enforced

### DISCUSSION THREAD

**Discussion by sgeofrank** [requarks/wiki#3724] [Error / Bug Report]
**Describe the bug**
After merging https://github.com/Requarks/wiki/commit/7d23344c7af80909c43e42a383df4d916f7eedf6, we continue to experience page rule problems. We created a group that can read the homepage, and has access to read/write pages under a different specific path. However, users can still edit the homepage, despite no pagerule to write/edit for that page.
![image](https://user-images.githubusercontent.com/23427609/74260024-d9d3e680-4cc6-11ea-82b2-8db36dd94751.png)

We cleared the content caches and set the JWT to 2 minutes and waited 10 minutes.

**To Reproduce**
1. Create a User Group with all content permissions.  
2. On page rules, create a rule for an 'exact page path' to read pages, view pages source, view pages, history, read/use assets, read comments, and create comments.
![image](https://user-images.githubusercontent.com/23427609/74260899-50bdaf00-4cc8-11ea-80a5-dc4fb31afbe4.png)
3. Create a second rule for a 'path start with' that doesn't encompass the page with the previous rule. Give read pages, create pages, edit + move page, view pages sourcce, view pages history, read/use assets, upload assets, edit + delete assets, read comments, create comments, edit + delete comments. 
![image](https://user-images.githubusercontent.com/23427609/74260953-67fc9c80-4cc8-11ea-87b0-e26ed8e9c702.png)
4. Create a new user in the group
5. Navigate to the page with the read only permissions.
6. Open page editor and edit the page. Save.

**Expected behavior**
When a page has read only permissions, I'd expect the page edit icon to be missing from the page. If it is present, I'd expect to get a permission warning before getting to the editor, saying I don't have permission. If I get to the page editor, I'd expect another check on save, that would throw a permission error.

**Screenshots**

**Host Info (please complete the following information):**
 - OS: your standard Heroku install
 - Wiki.js version: current as of commit: 9384ce1
 - Database engine: postgres 11.6

**Additional context**
We did intermittently hit a permission wall on an existing test user in the same group trying to edit the homepage. However, after 2 tests, that user now no longer hits the permission wall. We've tested on clean browsers, and via chrome incognito. We also tested on a clean machine that had no site data for our domain.

**Comment by MicBoucinha** [comment]
I can confirm this. We have a similar situation with the final v2.0.12. 
**Host Info (please complete the following information):**
- OS: Docker version
- Wiki.js version: v2.0.12
- Database engine: postgres 9 (default from the Docker image)

**Comment by MicBoucinha** [comment]
After further testing and upgrading to the latest final version, the page rules are working nicely.

**Comment by nwpray** [comment]
@nicolasg-playster this appears to be working as of 2.1.113 THANK YOU! :) You should be able to resolve this now.
